### PR TITLE
feat: add story for slice component

### PR DESCRIPTION
**Pull request by aronshamash**
Adds the most basic story for the `Slice` component.
Fixes dextrose issue due to the component not having a story.

## Desktop Chrome
screen shot 2018-02-16 at 10 30 56

### CODE REVIEW

File: packages/slice/slice.stories.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { View } from "react-native";
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import Slice from "./slice";
+
+const Module = props => (
```
Comment (colincclark) [inline packages/slice/slice.stories.js]: destruct colour in the argument

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/0a7e7471612b6d43cf16cb6c8316351a22e224ee/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/56a3f0572dc27ef667921f01c38b52b71f8b7ab5/dextrose-test/index.html

**Comment by colincclark** [comment]
This does look right. The Slice should handle this layout better, no?

screen shot 2018-02-16 at 09 43 08

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/9d912857aebe23f8b93ed459c82d021d784fc365/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f71711f5997b155e08b72c5d25061c602354265a/dextrose-test/index.html

**Pull request merged**